website/content/labs/advanced-git.md

```diff
@@ -299,7 +299,7 @@ the heads of the two branches. For example, in this diagram, commit `854bff54` m
 ![branches3](/cmsc22000/img/git-lab-branches-3.png)
 
 Our repository actually has two other branches: `change_author1` and
-`change_author2`. Go to https://github.com/GITHUB_USERNAME/cmsc22000-dummy-repo
+`change_author2`. Go to https://github.com/GITHUB_USERNAME/2020-advanced-git
```

Comment:
These branches don't exist in my forked repo: see https://github.com/dwahme/2020-advanced-git

Reply:
Ok, this should be fixed now (it looks like, when I migrated the previous repo we've used for this, it only migrated the master branch)